src/parsing/parsers/generic/jison/alter/alter_table.jison

```diff
@@ -26,6 +26,7 @@ AlterTable
  : AlterTableLeftSide PartitionSpec
  ;
 
+// TODO: support AlterTableRightSide, PartitionSpec will not work for MySQL, PostgreSQL
```

Comment:
We don't have `ALTER TABLE table_name ADD column_name datatype;` supported, sadly :(

Reply:
Yeah, noticed that..